tests/helpers/helper.go

```diff
@@ -0,0 +1,162 @@
+package helpers
+
+import (
+	"bytes"
+	"crypto/tls"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"time"
+
+	"github.com/rancher/wrangler/pkg/gvk"
+	"github.com/rancher/wrangler/pkg/unstructured"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/discovery"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/restmapper"
+	"k8s.io/kubernetes/pkg/apis/authentication"
+
+	"github.com/google/uuid"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/kubernetes/pkg/apis/admission"
+)
+
+// GenerateMutationRequest is a helper method to perform an admission review request
+// against a webhook and return the response to call
+func GenerateMutationRequest(source runtime.Object, endpoint string, cfg *rest.Config) (*admission.AdmissionResponse, error) {
```

Comment:
will this be a one-time request if the HTTP request is failed?

Reply:
if the http request is failed then the mutating webhook will ignore the failure and allow pod to be created, without the SYS_RESOURCE capability added